src/user-defined-types/const.md

```diff
@@ -0,0 +1,39 @@
+# `const`
+
+Constant variables are evaluated at compile time and their values are inlined
```

Comment:
One minor request: Could you change the wording here to not refer to constants as "variables"? That was a minor tweak I've been meaning to make, and it seems reasonable to do it here since you're reworking the pages already.

```suggestion
Constants are evaluated at compile time and their values are inlined
```